Project.toml

```
@@ -33,5 +33,5 @@ Static = "0.8, 1"
 StaticArrayInterface = "1.5.1"
 StaticArrays = "1"
 StrideArrays = "0.1.28"
-Trixi = "0.9.9, 0.10, 0.11, 0.12"
+Trixi = "0.9.9, 0.10, 0.11, 0.12, 0.13"
```

```suggestion
Trixi = "0.13"
```

Since this relies on the Adapt.jl PR and the switch of the default wave speeds, we have to update the compat settings accordingly.